src/main/java/com/mineboundteam/minebound/item/ArmorMaterials.java

```diff
@@ -15,9 +15,12 @@
 public enum ArmorMaterials implements ArmorMaterial {
     MYRIAL_EFFIGY(0, new int[]{1, 4, 5, 2}, 0, SoundEvents.ARMOR_EQUIP_CHAIN, 0f, 0f, null),
     MYRIAL_SUIT(0, new int[] { 2, 5, 6, 2 }, 0, SoundEvents.ARMOR_EQUIP_IRON, 0f, 0f, null),
-    MYRIAL_SYNERGY(0, new int[] { 3, 6, 8, 3 }, 0, SoundEvents.ARMOR_EQUIP_DIAMOND, 0f, 0f, null), 
-    ENERGIZED_IRON(15, new int[]{2, 5, 6, 2}, 8, SoundEvents.ARMOR_EQUIP_IRON, 0.5F, 0.0F,  Ingredient.of(ItemRegistry.ENERGIZED_IRON_INGOT.get())), 
-    ILLUMINANT_STEEL(30, new int[]{3, 5, 7, 2}, 8, SoundEvents.ARMOR_EQUIP_IRON, 1.0F, 0.0F, Ingredient.of(ItemRegistry.ILLUMINANT_STEEL_INGOT.get()));
+    MYRIAL_SYNERGY(0, new int[] { 3, 6, 8, 3 }, 0, SoundEvents.ARMOR_EQUIP_DIAMOND, 0f, 0f, null),
+    MYRIAL_SINGULARITY(0, new int[] { 3, 6, 8, 3 }, 0, SoundEvents.ARMOR_EQUIP_NETHERITE, 0f, 0f, null),
+    ENERGIZED_IRON(15, new int[]{2, 5, 6, 2}, 8, SoundEvents.ARMOR_EQUIP_IRON, 0.5F, 0.0F,  Ingredient.of(ItemRegistry.ENERGIZED_IRON_INGOT.get())),
+    ILLUMINANT_STEEL(30, new int[]{3, 5, 7, 2}, 8, SoundEvents.ARMOR_EQUIP_IRON, 1.0F, 0.0F, Ingredient.of(ItemRegistry.ILLUMINANT_STEEL_INGOT.get())),
+    GILDED_DIAMOND(33, new int[] { 3, 6, 8, 3 }, 8, SoundEvents.ARMOR_EQUIP_DIAMOND, 2.5F, 0,  Ingredient.of(ItemRegistry.GILDED_DIAMOND_INGOT.get())),
```

Comment:
make enchantability follow the tools and check those defense values. They shouldn't be copy and pasted. Both should also have some knockback resistance like .1 for gilded diamond and .3 for myrial. Myrial should also have higher toughness, like 3.5-4